velox/exec/fuzzer/AggregationFuzzerBase.cpp

```diff
@@ -239,7 +239,14 @@ std::vector<std::string> AggregationFuzzerBase::generateSortingKeys(
     std::vector<std::string>& names,
     std::vector<TypePtr>& types) {
   std::vector<std::string> keys;
-  auto numKeys = boost::random::uniform_int_distribution<uint32_t>(1, 5)(rng_);
+  uint32_t numKeys;
```

Comment:
This can be combined
```
uint32_t numKeys = 1;
if (vectorFuzzer_.coinToss(0.5)) {
  numKeys = boost::random::uniform_int_distribution<uint32_t>(2, 5)(rng_);
}
```
and does the same with less code.